### PR TITLE
Fixes #11191: Add a post-node-acceptation hook

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/11191

### CODE REVIEW

File: rudder-core/src/main/scala/com/normation/rudder/services/servers/NewNodeManager.scala [hunk 1]
```diff
@@ -145,6 +151,36 @@ trait NewNodeManager extends NewNodeManagerHooks {
...
+    val hookEnv = HookEnvPairs.build(
+        ("RUDDER_NODEID", nodeId.value)
+    )
+    val postHooksTime =  System.currentTimeMillis
+    HooksLogger.info(s"Executing post-node-acceptance hooks for node with id '${nodeId.value}'")
+    for {
+      postHooks     <- RunHooks.getHooks(HOOKS_D + "/node-post-acceptance", HOOKS_IGNORE_SUFFIXES)
+      runPostHook   =  RunHooks.syncRun(postHooks, hookEnv, systemEnv)
+      timePostHooks =  (System.currentTimeMillis - postHooksTime)
+      _             =  logger.debug(s"Node-post-deletion scripts hooks ran in ${timePostHooks} ms")
```
Comment (ncharles) [inline rudder-core/src/main/scala/com/normation/rudder/services/servers/NewNodeManager.scala]: Node-post-deletion

File: rudder-core/src/main/resources/hooks.d/node-post-acceptance/readme.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+= policy-generation-node-finished
```
Comment (ncharles) [inline rudder-core/src/main/resources/hooks.d/node-post-acceptance/readme.adoc]: shouldn't it be node-post-acceptance ?

**Comment by fanf** [comment]
I didn't wanted to put it in 4.1 because the feature is half-backed (no pre-node acceptance, not sure about the parameters passed to scripts) and I would prefer to have a real relection done on that subject before considering adding it to 4.1.

**Comment by fanf** [comment]
PR rebased

**Pull request merged**